migrations/v5.js

```diff
@@ -0,0 +1,127 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, getComponents, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-accordion - v4.0.0 > v5.0.0', async () => {
+  let course, courseAccordionGlobals, accordions;
+  const originalAriaRegion = 'Accordion. Select each button to expand the content.';
+  const newAriaRegion = 'List of expandable sections. Select each button to expand the content.';
+
+  whereFromPlugin('adapt-contrib-accordion - from v4.0.0', { name: 'adapt-contrib-accordion', version: '<5.0.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = getComponents('accordion');
+    return accordions.length;
+  });
+
+  mutateContent('adapt-contrib-accordion - modify globals ariaRegion attribute', async (content) => {
+    course = getCourse();
+    if (!_.has(course, '_globals._components._accordion.ariaRegion')) _.set(course, '_globals._components._accordion', { ariaRegion: originalAriaRegion });
+    courseAccordionGlobals = course._globals._components._accordion;
+    if (courseAccordionGlobals.ariaRegion === originalAriaRegion) {
+      courseAccordionGlobals.ariaRegion = newAriaRegion;
+    }
+    return true;
+  });
+
+  mutateContent('adapt-contrib-accordion - update accordion._supportedLayout attribute to full-width if current isn\'t a supported value.', async () => {
+    accordions.forEach(accordion => {
+      if (accordion._supportedLayout !== 'full-width' || accordion._supportedLayout !== 'half-width' || accordion._supportedLayout !== 'both') {
+        accordion._supportedLayout = 'full-width';
+      }
```

Comment:
Return early and testing multiple values against one variable:
```suggestion
      if (supportedLayouts.includes(accordion._supportedLayout)) return;
      accordion._supportedLayout = 'full-width';
```